app/src/main/res/values/array.xml

```diff
@@ -3,6 +3,14 @@
     xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="MissingTranslation">
 
+    <string-array name="app_theme_entries">
+        <item>@string/pref_theme_dark</item>
+        <item>@string/pref_theme_emerald</item>
+    </string-array>
+    <string-array name="app_theme_values">
```

Comment:
You forgot to update these with the enum change. (They should be exactly the same as the enums)